cmd/cli/generate/generate.go

```diff
@@ -0,0 +1,23 @@
+package generate
+
+import (
+	"crypto/ecdsa"
+	"github.com/nextdotid/proof-server/validator/ethereum"
+	"github.com/nextdotid/proof-server/validator/github"
+	"github.com/nextdotid/proof-server/validator/keybase"
+	"github.com/nextdotid/proof-server/validator/twitter"
+)
+
+func init_validators() {
+	twitter.Init()
+	github.Init()
+	keybase.Init()
+	ethereum.Init()
+}
+
+func GeneratePayload() {
+}
+
+func generate_ethereum(sign_payload string, persona_sk *ecdsa.PrivateKey) {
+
+}
```

Comment:
Don't put this module into `/cmd/`, since it's a special directory for go.

Move this `cli` package out of `/cmd/`.